### DISCUSSION TITLE
Backdrop Filter blur bug on google chrome

### DISCUSSION THREAD

**Discussion by fiqryq** [tailwindlabs/tailwindcss#6683] [Help]
**What version of Tailwind CSS are you using?**

``` I use v2.2.16 version ```

**What build tool (or framework if it abstracts the build tool) are you using?**

```
- Postcss I use this when install  npm install -D tailwindcss@latest postcss@latest autoprefixer@latest
- Nextjs 12
  ```

**What version of Node.js are you using?**

```
I use version 16.10.0
```
**What browser are you using?**

```
- chrome
- firefox
- safari
```

**What operating system are you using?**

```
MacOS Monterey 
```

**Reproduction URL**

```html
<header className="flex items-center w-full justify-between py-5 sticky left-0 right-0 top-0 z-20 bg-white dark:bg-black bg-opacity-30 dark:bg-opacity-30 backdrop-filter backdrop-saturate-150 backdrop-blur-lg firefox:bg-opacity-100 dark:firefox:bg-opacity-100">

````
Link : [Repo](https://github.com/fiqryq/Fiqrychoerudin.dev/blob/11fa3931adad9289547a59b4c1f82ce6b835b16e/components/LayoutWrapper.tsx#L17)

**Describe your issue**
 when I use the backdrop filter what happens is that my navigation gets an error but it only happens in chrome browser.

https://user-images.githubusercontent.com/25787603/147027029-68bae4c3-f0e0-4a01-85f3-c129c0869a86.mov

**Comment by programVeins** [comment]
Facing a similar issue with nested backdrop blurs. It seems like this is a chromium issue, you might wanna check this reddit thread: https://www.reddit.com/r/Frontend/comments/xm2ft0/comment/ipm7hh9/?utm_source=share&utm_medium=web2x&context=3

**Reply by Metehan-Altuntekin** [reply]
Thanks, this was the problem in my case
